src/CustomerProfile.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState, useEffect } from "react";
+
+export default function CustomerProfile({ customerId }) {
+  const [customerInfo, setCustomerInfo] = useState({});
+  useEffect(() => {
+    console.log("...");
+    fetch(`https://cyf-react.glitch.me/customers/${customerId}`)
+      .then(res => res.json())
+      .then(data => setCustomerInfo(data));
+  }, [customerId]);
+  return (
+    <div>
+      {customerId === 0 ? null : (
```

Comment:
if you are just wanting to display something when a statement equates to true, you can use `&&` in react, for example:
```
{customerId && (
//content you want to render
```
mentioned above but I would use null for your customerId rather than 0, setting as 0 is a bit confusing & can lead to problems if a customer does have an id of 0